x/examples/website-wrapper-app/package.json

```
@@ -21,8 +21,7 @@
     "open:android": "cd output/wrapper_app_project && npm run open:android",
     "open:ios": "cd output/wrapper_app_project && npm run open:ios",
     "reset": "npm run clean && npm ci",
-    "start:navigator": "node ./basic_navigator_example/.scripts/start.mjs",
-    "start": "node ./.scripts/start.mjs"
+    "start": "node ./basic_navigator_example/.scripts/start.mjs"
```

To make it clear this is not starting the app:
```suggestion
    "navigator:start": "node ./basic_navigator_example/.scripts/start.mjs"
```